### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Common questions_ScreenReader: Status message not announced for the Thank you text that is been displayed after we submit the feedback for the page.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3586]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS4.1.3#Win10;#Visual Studio Code;#CELAP1_code;#Narrator;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)
Screen Reader: Narrator(Desktop App)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/versioncontrol
2)Using Version Control in VS Code Page will get open.
3)Go through "Common questions" heading till the "Was this documentation helpful?" section at the bottom of the page.

**Actual:**
Status message is not announced by the screen reader when the user submits the feedback for the document by selecting yes/no button and providing details on feedback if necessary.

Similar issue has been observed in following pages:
1.https://code.visualstudio.com/docs/editor/whyvscode
2.https://code.visualstudio.com/docs/getstarted/settings
3.https://code.visualstudio.com/docs/python/python-tutorial
4.https://code.visualstudio.com/docs/getstarted/tips-and-tricks
5.https://code.visualstudio.com/docs/setup/linux
6.https://code.visualstudio.com/docs/editor/integrated-terminal

**Expected:**
Status message regarding the thank you text that appears on the screen after feedback is submitted should be announced by screen reader.

**User Impact:**
Screen Reader users might not come to know about the content that has been updated after submitting the feedback for the page.

**Attachment link for Reference:**
[3586_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Common questions_ScreenReader.zip](https://github.com/microsoft/vscode-docs/files/4510973/3586_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_Common.questions_ScreenReader.zip)

**Comment by gregvanl** [comment]
Removed the thank you message fade out so the text stays on the page for screen readers.
